Threads/src/main/java/task1/Card.java

```diff
@@ -0,0 +1,37 @@
+package task1;
+
+
+public class Card {
+
+    private StringBuffer name;
```

Comment:
lets collect guests to any structure (maybe list) instead of append names to one string.